content/ember-4-0-released.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Ember 4.0 released
+authors:
+  - jen-weber
+date: 2021-11-22T00:00:00.000Z
+tags:
+  - releases
+  - '2021'
+  - version-4-x
+---
+
+
+Today, the Ember project is releasing version 4.0.0 of Ember.js, Ember Data, and
+Ember CLI. Ember it focuses
+the framework by removing long-deprecated APIs and support for legacy platforms.
+With these changes, Ember Octane becomes the norm!
+As planned, this release introduces no new features.
```

Comment:
This is orthogonal to the release post, but I feel like we need a subclass of semver to communicate our sort of take on this. Some projects just do breaking changes whenever the feel like and never deprecate things, and they burn through majors like it's their job. :thinking: Maybe @wycats' wordsmithery has some ideas <3

Reply:
(though, I now see this is expanded on in the next section, but good words give good marketing, so :shrug: )

Reply:
That's a great point. I think we need a full blog post on this topic. We just need a name for it. I don't think there is one.

Reply:
We should be careful to keep the release post focused on the release. It is dense enough as is, and imo not the place to argue or advocate for release/versioning strategy in depth.

Reply:
I agree with @mixonic - @NullVoxPopuli if you want to DM me in Discord, I'll help you coordinate this. I'm going to resolve this conversation for now.